lib/temporal/client.rb

```diff
@@ -59,6 +94,14 @@ def start_workflow( # rubocop:disable Metrics/ParameterLists
       implementation.start_workflow(input)
     end
 
+    # Get a workflow handle to an existing workflow by its ID.
+    #
+    # @param id [String] Workflow ID to get a handle to.
+    # @param run_id [String, nil] Run ID that will be used for all calls.
+    # @param first_execution_run_id [String, nil] First execution run ID used for cancellation and
```

Comment:
Not sure if this is the place to explain first_execution_run_id but I'd at least mention that this refers to the first in a chain of workflows, workflows are chained when using continue-as-new, retries (as permitted by the retry policy) and cron executions.

Reply:
Copied from Python SDK, but happy to update